### ISSUE TITLE
Change symbol "Π" to always be "π" and change your extended popups

### ISSUE THREAD

**Issue by RickyM7** [florisboard/florisboard#298]
I just realized that the symbol `π` turns into `Π` when in capslock. Could you change this so that the `π` symbol remains independent of the capslock state? Another thing, the extended popups of `π` are `ω` and `μ`, and those of `Π` are `Ω` and `Μ`(But the `M` can be removed because it doesn't even make sense that it's an extended popup of the symbol `π`), could you add the extended popups of `Π` to `π`?

In short: The symbol `π` remains regardless of the state of the capslock (just like any other symbol it is) and its extended popups will be `Ω`, `Π`, `ω` and `μ`.

**Comment by patrickgold** [comment]
Thanks for the suggestion! You are right, the capitalization of pi is indeed a byproduct of me not being aware enough that the caps from the character keyboard also influences the symbols layout. I will add this on my todo list to fix this!

**Comment by patrickgold** [comment]
Above commit prevents the capitalization of any key outside the character layout scope. The `pi` key now has these popups (independent of the caps state): (alpha and beta have additionally to your request been added as I needed two additional popup letters and these are the most commonly used ones besides the already included Greek letters)

![pi_popups](https://user-images.githubusercontent.com/19412843/106677876-dad9e400-65b9-11eb-8085-d9ccd2bf432d.jpg)

Will be released in the next version v0.3.7. Thanks again for reporting!

**Comment by RickyM7** [comment]
Perfect, thank you!

**Comment by RickyM7** [comment]
@patrickgold I just saw that this bug is occurring again in v0.3.9.

**Comment by kj7rrv** [comment]
Confirmed on my Moto G Power 2020.On Mar 17, 2021 12:29, Ricardo ***@***.***> wrote:
@patrickgold I just saw that this bug is occurring again in v0.3.9.

**Comment by kj7rrv** [comment]
(v0.3.9 [technically v0.3.10beta01])On Mar 17, 2021 12:32, Sam Sloniker ***@***.***> wrote:Confirmed on my Moto G Power 2020.On Mar 17, 2021 12:29, Ricardo ***@***.***> wrote:
@patrickgold I just saw that this bug is occurring again in v0.3.9.

**Comment by patrickgold** [comment]
Well if I remove the safeguard (93654c4f88f9ebb41b459a8c1854e133e03bea26) that caps only works in character layouts it then this bug will of course occur again... Will add it again momentarily.

**Comment by patrickgold** [comment]
Above commit fixes the uppercase bug, hopefully the last time :)

**Comment by RickyM7** [comment]
@patrickgold Thank you very much, I just tested the v0.3.10-beta2 and the bug has been fixed. I'm sorry I haven't tested it before, it's just that my internet was terrible these last 5 days.
